src/components/Beer.vue

```diff
@@ -57,22 +59,60 @@
 </template>
 
 <style scoped lang="scss">
+@font-face {
+  font-family: "Signboard";
+  font-style: normal;
+  font-weight: 400;
+  src: url(../assets/fonts/signboard.woff2);
```

Comment:
when you add css classes for some of the comments above, can you include the sources in those as well?

Reply:
What do you mean by this?